applications/RANSApplication/custom_utilities/rans_calculation_utilities.cpp

```diff
@@ -52,6 +53,35 @@ void CalculateGeometryData(
     }
 }
 
+void CalculateConditionGeometryData(
+    const GeometryType& rGeometry,
+    const GeometryData::IntegrationMethod& rIntegrationMethod,
+    Vector& rGaussWeights,
+    Matrix& rNContainer)
+{
+    const GeometryType::IntegrationPointsArrayType& integration_points =
+        rGeometry.IntegrationPoints(rIntegrationMethod);
+
+    const std::size_t number_of_integration_points = integration_points.size();
+    const int dimension = rGeometry.WorkingSpaceDimension();
+    const double domain_size = rGeometry.DomainSize();
+
+    if (rGaussWeights.size() != number_of_integration_points)
+    {
+        rGaussWeights.resize(number_of_integration_points, false);
+    }
+
+    rNContainer = rGeometry.ShapeFunctionsValues(rIntegrationMethod);
+
+    // CAUTION: "Jacobian" is 2.0*A for triangles but 0.5*A for lines
+    double det_J = (dimension == 2) ? 0.5 * domain_size : 2.0 * domain_size;
+
+    for (unsigned int g = 0; g < number_of_integration_points; g++)
+    {
+        rGaussWeights[g] = det_J * integration_points[g].Weight();
+    }
+}
```

Comment:
I think this should go to the `GeometryUtils` as a new method that does not compute the gradients.

Reply:
I agree, lets do it in a different PR. :)